pages/en/lb4/Getting-started.md

```diff
@@ -24,28 +24,78 @@ The starting point is [ECMAScript 6](http://www.ecma-international.org/ecma-262/
 - Running the app (node ., npm start)
 " %}
 
+## Installation
+
+Make sure you have the following installed:
+
+- Node.js >= 7.0.0
+- TypeScript >= 2.0.0 `npm i -g typescript`
+- TypeScript Node >= 3.0.0 `npm i -g ts-node`
+
 ## A simple example
 
-Here is the most basic LoopBack 4 application:
+This example creates an `Application` that responds to all HTTP requests with the text "Hello World".
+
+### Building a LoopBack 4 application
+First create a new folder.  Then, create two files within this folder: `index.ts` and `tsconfig.json`.  
 
+{% include code-caption.html content="index.ts" %}
 ```js
 import {Application} from '@loopback/core';
+import {RestComponent, RestServer} from '@loopback/rest';
 
-const app = new Application();
-app.handler((sequence, request, response) => {
-  sequence.send(response, 'hello world');
-});
-
-(async function start() {
-  await app.start();
-  console.log(`The app is running on port ${app.getSync('http.port')}`);
+export class HelloWorldApp extends Application {
+    constructor() {
+      super({
+        components: [RestComponent]
+      });
+    }
+  
+    async start() {
+      const rest = await this.getServer(RestServer);
+      rest.handler((sequence, request, response) => {
+        sequence.send(response, 'Hello World');
+      });
+      await super.start();
+      console.log(`REST server running on port: ${await rest.get('rest.port')}`);
+    }
+  }
+  
+(async function main() {
+    const app = new HelloWorldApp();
+    await app.start();
 })();
+
 ```
 
-The example above creates an `Application` that responds to all HTTP requests with the text "Hello World".
+Next, create a file called tsconfig.json to indicate that this is a TypeScript project and specify the TypeScript compiler options required.  For more details, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+{% include code-caption.html content="tsconfig.json" %}
+```
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "es6",                          
```

Comment:
Let's ask people to install the latest Node.js 8.x and then use `"target": "es2017"`. That way we can leverage native async/await.